charts/merlin/README.md

```diff
@@ -110,36 +208,13 @@ The following table lists the configurable parameters of the Merlin chart and th
 | environmentConfigs[0].namespace_timeout | string | `"2m"` |  |
 | environmentConfigs[0].queue_resource_percentage | string | `"20"` |  |
 | environmentConfigs[0].region | string | `"id"` |  |
-| feastCoreApi.apiHost | string | `"http://feast-core.mlp:8080/v1"` |  |
 | global.protocol | string | `"http"` |  |
-| imageBuilder.baseImages."3.7.*".buildContextURI | string | `"git://github.com/gojek/merlin.git#refs/tags/v0.1"` |  |
-| imageBuilder.baseImages."3.7.*".dockerfilePath | string | `"docker/Dockerfile"` |  |
-| imageBuilder.baseImages."3.7.*".imageName | string | `"pyfunc-py37:v0.1.0"` |  |
-| imageBuilder.baseImages."3.7.*".mainAppPath | string | `"/merlin-spark-app/main.py"` |  |
 | imageBuilder.clusterName | string | `"test"` |  |
-| imageBuilder.dockerRegistry | string | `"dockerRegistry"` |  |
-| imageBuilder.k8sConfig | string | `""` |  |
-| imageBuilder.kanikoImage | string | `"gcr.io/kaniko-project/executor:v1.6.0"` |  |
-| imageBuilder.maxRetry | int | `3` |  |
-| imageBuilder.namespace | string | `"mlp"` |  |
-| imageBuilder.nodeSelectors | object | `{}` |  |
-| imageBuilder.predictionJobBaseImages."3.7.*".buildContextURI | string | `"git://github.com/gojek/merlin.git#refs/tags/v0.1"` |  |
-| imageBuilder.predictionJobBaseImages."3.7.*".dockerfilePath | string | `"docker/app.Dockerfile"` |  |
-| imageBuilder.predictionJobBaseImages."3.7.*".imageName | string | `"pyspark-py37:v0.1.0"` |  |
-| imageBuilder.predictionJobBaseImages."3.7.*".mainAppPath | string | `"/merlin-spark-app/main.py"` |  |
-| imageBuilder.predictionJobContextSubPath | string | `""` |  |
-| imageBuilder.resources.limits.cpu | string | `"1"` |  |
-| imageBuilder.resources.limits.memory | string | `"1Gi"` |  |
-| imageBuilder.resources.requests.cpu | string | `"1"` |  |
-| imageBuilder.resources.requests.memory | string | `"512Mi"` |  |
-| imageBuilder.retention | string | `"48h"` |  |
-| imageBuilder.safeToEvict | bool | `false` |  |
+| imageBuilder.k8s_config | object | `{}` |  |
```

Comment:
Kinda ugly that we have this and also `config.ImageBuilderConfig`. This is also the same for Turing. We should refactor this later.

Reply:
Yep agreed, another way to see it is to imagine that `imageBuilder.k8s_config` houses all the image builder **_cluster_**-related configs while `config.ImageBuilderConfig` houses all the other image-builder configs. 

Alternatively, I can move all the configs in `config.ImageBuilderConfig` to fields under `imageBuilder`. What do you think?

Reply:
> Alternatively, I can move all the configs in config.ImageBuilderConfig to fields under imageBuilder

This would be nice. But simultaneously, it would be good to ensure some level of flexibility if the configs change over time. So, can we do something like:

```
imageBuilder:
    serviceAccount: {}
    ...
    config:
        k8s_config: {}
    ....
```

and mostly just just `config.ImageBuilderConfig` to `imageBuilder.config`, minimally manipulating things like the service account?

Reply:
Hmm I was thinking of keeping the `k8s_config` field at thee same levels as `serviceAccount`, just so it remains the same as in Turing. Then perhaps create a new `builderConfig` field where we'll store everything else from `config.ImageBuilderConfig`.

Reply:
Ok, that works too.